File: matchteam/src/main/java/com/example/matchteam/dto/board/FindBoardDto.java
package com.example.matchteam.dto.board;

import lombok.Getter;
import lombok.Setter;

import java.sql.Timestamp;

@Getter
@Setter
public class FindBoardDto {
    private String title;
    private String content;
    private Timestamp createdDate;
    private int viewCnt;
}

File: MatchingTeam/app/src/main/java/com/example/matchingteam/dto/board/FindBoardDto.kt
package com.example.matchingteam.dto.board

import com.google.gson.annotations.SerializedName
import java.sql.Timestamp

data class FindBoardDto(
    @SerializedName("title") val title: String,
    @SerializedName("content") val content: String,
    @SerializedName("createdDate") val createdDate: Timestamp,
    @SerializedName("viewCnt") val viewCnt: Int
)
